src/pages/trial-end-mail.html

```diff
@@ -12,13 +12,7 @@
             <spacer size="24"></spacer>
 
             <p  class="body-1 c-black m-0">
-                Your {{{{raw}}}}{{var:subscription_type}}{{{{/raw}}}} trial <b class="txt-m">ends in 3 days.</b>
-            </p>
-
-            <spacer size="12"></spacer>
-
-            <p  class="body-1 c-black m-0">
-                Moving forward you will be charged automatically on a {{{{raw}}}}{{var:subscription_duration}}{{{{/raw}}}} basis according to your plan.
+                Your {{var:subscription_type}} subscription will be renewed on {{var:charge_date}} and your card will be charged.
```

Comment:
`trial-end` should stay the same. Only the text `automatic_renewal` should change since it mentions trial and doesn't fit the case